torchio/transforms/augmentation/intensity/random_labels_to_image.py

```diff
@@ -0,0 +1,222 @@
+        >>> transform = RandomLabelsToImage(label_key='cls')
+        >>> # Using custom gaussian_parameters
+        >>> label_values = colin['cls'][DATA].unique().round()
+        >>> gaussian_parameters = {
+        >>>     int(label): {
+        >>>         'mean': i / len(label_values), 'std': 0.01
+        >>>     } for i, label in enumerate(label_values)
+        >>> }
+        >>> transform = RandomLabelsToImage(label_key='cls', gaussian_parameters=gaussian_parameters)
+        >>> # Filling holes of the simulated image with the original T1 image
+        >>> rescale_transform = RescaleIntensity((0, 1), (1, 99))   # Rescale intensity before filling holes
+        >>> simulation_transform = RandomLabelsToImage(
+        >>>     label_key='cls',
+        >>>     image_key='t1',
+        >>>     gaussian_parameters={0: {'mean': 0, 'std': 0}}
+        >>> )
+        >>> transform = Compose([rescale_transform, simulation_transform])
+        >>> transformed = transform(colin)  # colin has a new key 'image' with the simulated image
+    """
+    def __init__(
+            self,
+            label_key: Optional[str] = None,
+            pv_label_keys: Optional[Sequence[str]] = None,
+            image_key: str = 'image',
+            gaussian_parameters: Optional[Dict[Union[str, TypeNumber], Dict[str, TypeRangeFloat]]] = None,
+            default_gaussian_parameters: Optional[Dict[str, TypeRangeFloat]] = None,
+            binarize: bool = False,
+            p: float = 1,
+            seed: Optional[int] = None,
+            ):
+        super().__init__(p=p, seed=seed)
+        self.check_keys(label_key, pv_label_keys)
+        self.label_key = label_key
+        self.pv_label_keys = pv_label_keys
+        self.default_gaussian_parameters = self.parse_default_gaussian_parameters(default_gaussian_parameters)
+        self.gaussian_parameters = self.parse_gaussian_parameters(gaussian_parameters)
+        self.image_key = image_key
+        self.binarize = binarize
+
+    @staticmethod
+    def check_keys(label_key, pv_label_keys):
+        if label_key is not None and pv_label_keys is not None:
+            raise ValueError('"label_key" and "pv_label_keys" can\'t be set at the same time.')
+        if label_key is None and pv_label_keys is None:
+            raise ValueError('One of "label_key" and "pv_label_keys" must be set.')
+
+    def apply_transform(self, sample: Subject) -> dict:
+        random_parameters_images_dict = {}
+        original_image = sample.get(self.image_key)
+
+        if self.pv_label_keys is not None:
+            label_map = torch.cat([sample[key][DATA] for key in self.pv_label_keys], dim=0)
+            affine = sample[self.pv_label_keys[0]][AFFINE]
+            n_labels, *image_shape = label_map.shape
+            labels = self.pv_label_keys
+            values = list(range(n_labels))
+
+            if self.binarize:
+                # Remove values where all labels are 0
```

Comment:
put this comment one line after,
and precise here (and also in the doc) that binarization is done here by taking the bigger PV if different class are present in a voxel